x/pocketcore/types/service.go

```diff
@@ -152,7 +154,8 @@ type Payload struct {
 func (p Payload) Bytes() []byte {
 	bz, err := json.Marshal(p)
 	if err != nil {
-		panic(fmt.Sprintf("an error occured converting the payload to bytes:\n%v", err))
+		fmt.Println(fmt.Errorf("an error occured converting the payload to bytes:\n%v", err))
+		os.Exit(1)
```

Comment:
```suggestion
  log.Fatalf("an error occured converting the payload to bytes:\n%v", err)
```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)